runtime/compiler/runtime/Runtime.cpp

```diff
@@ -616,8 +616,8 @@ JIT_HELPER(_nativeStaticHelper);
 JIT_HELPER(jitLookupInterfaceMethod);
 JIT_HELPER(jitMethodIsNative);
 JIT_HELPER(jitMethodIsSync);
-JIT_HELPER(jitPreJNICallOffloadCheck);
-JIT_HELPER(jitPostJNICallOffloadCheck);
+JIT_HELPER(fast_jitPreJNICallOffloadCheck);
+JIT_HELPER(fast_jitPostJNICallOffloadCheck);
```

Comment:
Is there a reason for deviation from the naming convention?

Reply:
I changed this to use the same name as the VM helper name for consistency. Didn't realize that the JIT helpers have a naming convention. I'll change it back.

Reply:
I'm keeping the `fast_` in the names because these two lines are effectively `extern c fast_*` and referring to the VM helpers.